pom.xml

```diff
@@ -124,28 +125,8 @@
 							<version>2.1.0-SNAPSHOT</version>
 						</artifact>
 					</target>
-					<environments>
-						<environment>
-							<os>win32</os>
-							<ws>win32</ws>
-							<arch>x86_64</arch>
-						</environment>
-						<environment>
-							<os>linux</os>
-							<ws>gtk</ws>
-							<arch>x86_64</arch>
-						</environment>
-						<environment>
-							<os>macosx</os>
-							<ws>cocoa</ws>
-							<arch>x86_64</arch>
-						</environment>
-						<environment>
-							<os>macosx</os>
-							<ws>cocoa</ws>
-							<arch>aarch64</arch>
-						</environment>
-					</environments>
+					<!-- don't define any environments to let Tycho automatically pick the current one -->
```

Comment:
IIRC this is discouraged.
Can you explain why this does not cause problems on Linux (where three out of the four envs don't match as well), but does on Mac?

Reply:
There is a lengthy discussion around this in https://github.com/eclipse-m2e/m2e-core/discussions/1153. Probably a bug in Tycho, but I don't see an issue right now without environments....

Reply:
As https://github.com/eclipse-tycho/tycho/issues/1900 has been fixed I restored the environments again.

Reply:
Great. 👍🏽